packages/calling-component-bindings/src/baseSelectors.ts

```diff
@@ -21,8 +28,33 @@ export const getIncomingCallsEnded = (state: CallClientState): IncomingCallState
 
 export const getDeviceManager = (state: CallClientState): DeviceManagerState => state.deviceManager;
 
-export const getCall = (state: CallClientState, props: CallingBaseSelectorProps): CallState | undefined =>
-  state.calls[props.callId];
+export const getCallExists = (state: CallClientState, props: CallingBaseSelectorProps): boolean =>
+  !!state.calls[props.callId];
+
+export const getRemoteParticipants = (
+  state: CallClientState,
+  props: CallingBaseSelectorProps
+):
+  | undefined
+  | {
+      [keys: string]: RemoteParticipantState;
+    } => state.calls[props.callId]?.remoteParticipants;
+
+export const getIsScreenSharingOn = (state: CallClientState, props: CallingBaseSelectorProps): boolean =>
+  state.calls[props.callId]?.isScreenSharingOn ?? false;
```

Comment:
The base selectors should cheaply pull out values from the state. They should not contain any other business logic, in particular default values.

Instead of defaulting to `false`, I'd recommend returngin `boolean | undefined` and dealing with undefined values in the caller.

Same for `getIsMuted`.